src/web/js/trove/world.js

```diff
@@ -609,16 +706,40 @@
           var fixN = jsnums.toFixnum(n);
           return runtime.makeOpaque(new OnTick(handler, fixN * 1000));
         }),
+
+        "_patch_on-tick": makeFunction(function(handler, n) {
+          runtime.ffi.checkArity(arguments.length <= 1? 1: 2, arguments, "_patch_on-tick", false);
+          runtime.checkFunction(handler);
+          var fixN;
+          if (arguments.length >= 2) {
+            fixN = typeof n === "number"? n: n.toFixnum();
+          } else {
+            fixN = DEFAULT_TICK_DELAY;
+          }
+          return runtime.makeOpaque(new OnTick(handler, fixN * 1000));
+        }),
+
         "to-draw": makeFunction(function(drawer) {
           runtime.ffi.checkArity(1, arguments, "to-draw", false);
           runtime.checkFunction(drawer);
           return runtime.makeOpaque(new ToDraw(drawer));
         }),
+        "on-redraw": makeFunction(function(drawer) {
```

Comment:
What is this handler for?

Reply:
This one is almost certainly for compatibility with really old WeScheme programs. The design goal of Patch is that any WeScheme program that ran before should continue to run, and some Very Old WeScheme programs used `on-redraw` instead of `to-draw`

Reply:
This seems like something perhaps a Patch-specific wrapper around world.js might achieve, rather than baking it into the underlying world.js library that's exposed to Pyret?